potato/create_task_cli.py

```diff
@@ -136,56 +170,73 @@ def create_task_cli():
 
         atype = get_annotation_type()
 
-        desc = str(input('What description/question/instructions should annotators see for this?\n'))
-        name = str(input('What is the internal name for this category (used in output files)?\n'))        
-        
-        if atype == 'likert' or atype == 'bws':
-            min_label = ''
-            max_label = ''
-
-            if atype == 'likert':
-                size = str(input('How many items are on the likert scale?\n'))
+        desc = str(
+            input(
+                "What description/question/instructions should annotators see for this?\n"
+            )
+        )
+        name = str(
+            input(
+                "What is the internal name for this category (used in output files)?\n"
+            )
+        )
+
+        if atype == "likert" or atype == "bws":
+            min_label = ""
+            max_label = ""
+
+            if atype == "likert":
+                size = str(input("How many items are on the likert scale?\n"))
             else:
-                size = str(input('How many items to dislay at once for Best-Worst Scaling?\n'))
+                size = str(
+                    input(
+                        "How many items to dislay at once for Best-Worst Scaling?\n"
+                    )
+                )
+
```

Comment:
I really don't understand why Black is making this particular kind of change. Can we turn off certain flags for style differences?

Reply:
Done -- simply ran `black --line-length 100`. I think PEP 8 recommends line length of 80 characters, which is what Black was using. Anything longer than 100 characters should probably be broken up...